### PR TITLE
add Fairmode statistic to aeroval output

### LINKED ISSUE #601: CAMS2_83 FAIRMODE metrics in pyaerocom
implement FAIRMODE metrics in pyaerocom.
Daniel has already implemented MQI.
Next in line is: MQI90. That's the 90th percentile among all stations' MQIs.
90% of all stations have an MQI that is better (i.e. smaller) than MQI90.

After MQI90, we also need the three different parts of MQI. The bias indicator, standard deviation indicator, and correlation indicator.

Should have this in April.

### PR DESCRIPTION
**Pull request by avaldebe**
close #601

### CODE REVIEW

File: pyaerocom/aeroval/coldatatojson_helpers.py [hunk 1]
```diff
@@ -4,13 +4,16 @@
 import logging
 import os
 from datetime import datetime
+from math import sqrt
+from typing import Tuple
```
Comment (avaldebe) [inline pyaerocom/aeroval/coldatatojson_helpers.py:8]: unused imports?

File: pyaerocom/aeroval/coldatatojson_helpers.py [hunk 2]
```diff
@@ -1341,3 +1358,81 @@ def _init_data_default_frequencies(coldata, to_ts_types):
...
+    fairmode_stats["RMSU"] = rmsu
+    fairmode_stats["sign"] = sign
+    fairmode_stats["crms"] = crms
+    fairmode_stats["bias"] = bias
+    fairmode_stats["rms"] = rms
+    fairmode_stats["alpha"] = alpha
+    fairmode_stats["UrRV"] = UrRV
+    fairmode_stats["RV"] = RV
+
+    return fairmode_stats
```
Comment (avaldebe) [inline pyaerocom/aeroval/coldatatojson_helpers.py:1438]: duplicated code, see `fairmode_stats.py`

**Review by avaldebe — COMMENTED** [review]
some small things to clean up

**Review by dulte — APPROVED** [review]
ready to merge

**Pull request merged**